cmd/oras/pull.go

```diff
@@ -135,7 +135,7 @@ func runPull(opts pullOptions) error {
 				}
 				// Skip s if s is unnamed and has no successors.
 				if len(ss) == 0 {
-					if _, loaded := printed.LoadOrStore(s.Digest.String(), true); !loaded {
+					if _, loaded := printed.LoadOrStore(display.Name(s), true); !loaded {
```

Comment:
@wangxiaoxuan273 Will `desc.Digest.String() + desc.Annotations[ocispec.AnnotationTitle]` work?

Reply:
Yes it worked. Replaced with `desc.Digest.String() + desc.Annotations[ocispec.AnnotationTitle]`.